src/System.Diagnostics.DiagnosticSource/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,2 @@
+MembersMustExist : Member 'System.Diagnostics.DiagnosticListener.DefaultListener.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
The DefaultListener and DefaultSource properties were removed as part of API review and were never checked in, but the reference assembly was never updated.    I have a pull request #6817 pending to make the reference assembly match the implementation assembly.